platforms/android/tangram/src/main/java/com/mapzen/tangram/NativeLibraryLoader.java

```diff
@@ -0,0 +1,16 @@
+package com.mapzen.tangram;
+
+public class NativeLibraryLoader {
+    public static final boolean sNativeLibraryLoaded = loadLibrary();
+
+    private static synchronized boolean loadLibrary() {
+        try {
+            System.loadLibrary("c++_shared");
+            System.loadLibrary("tangram");
+            return true;
+        } catch (final Exception e) {
```

Comment:
This should probably be more specific than "any exception". The expected exceptions here would be `SecurityException` or `UnsatisfiedLinkError` (https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#loadLibrary(java.lang.String))